views/name_form.erb

```diff
@@ -0,0 +1,5 @@
+<form action="/yourname" method="post">
+  <input type="text" name="name">
+  <input type="submit" value="Submit your name">
+</form>
+
```

Comment:
There are 3 .ERB files. Wondering if it is recommended to have just 1 .ERB file as has been the practice in the previous excercises